CONTRIBUTING.md

```diff
@@ -2,37 +2,37 @@
 
 Contributors are asked to provide a Contributor License Agreement for working on the project:
 
-* [corporate_contributor.txt](http://www.osgeo.org/sites/osgeo.org/files/Page/corporate_contributor.txt)
-* [individual_contributor.txt](http://www.osgeo.org/sites/osgeo.org/files/Page/individual_contributor.txt)
+* [Corporate contributor license](https://www.osgeo.org/resources/corporate-contributor-license/)
+* [Individual contributor license](https://www.osgeo.org/resources/individual-contributor-license/)
 
-This agreement can be printed, signed, scanned, and emailed to [info@osgeo.org](mailto:info@osgeo.org) at the Open Source Geospatial Foundation (OSGeo). [OSGeo](http://www.osgeo.org/content/foundation/about.html)
+This agreement can be printed, signed, scanned, and emailed to [info@osgeo.org](mailto:info@osgeo.org) at the Open Source Geospatial Foundation (OSGeo). [OSGeo](https://www.osgeo.org/about/)
```

Comment:
Think the email contact may need to be secretary@osgeo.org? Nope info@osgeo.org is correct.